schedule_lessons/dashboard/static/dashboard/js/agenda.js

```diff
@@ -1,6 +1,32 @@
 $(document).ready(function () {
   $('[data-toggle="tooltip"]').tooltip();
 
+  $(".acknowledge-btn").on('click', function(e) {
+    var data;
+    if (!$('#dontShowTutorialCheckbox').is(":checked")) {
+      data = {
+        'show_tutorial': 1, // true
+      }
+    } else {
+      data = {
+        'show_tutorial': 0, // false 
+      };
+    }
+    $(".tutorial-wrapper").remove();
+    $.ajax({
+      url: '/dashboard/save_tutorial_preferences',
+      type: 'post',
+      data: data,
+      error: function(xhr, status) {
```

Comment:
are these error and success callbacks mandatory to define if we aren't even doing anything in them??

I would say remove the success one for now, and in the error one put like an error message that gets displayed to user with the error returned from backend as an alert or something?

Reply:
it kinda looks like the backend doesn't really error on this endpoint, so maybe just remove both cases then for now?

Reply:
Addressed both concerns above with latest commit. Let me know if there is anything else you want improved. Thanks for the input.